Source/Core/Core/Config/MainSettings.cpp

```diff
@@ -280,6 +280,7 @@ const Info<std::string> MAIN_AUDIO_BACKEND{{System::Main, "DSP", "Backend"},
                                            AudioCommon::GetDefaultSoundBackend()};
 const Info<int> MAIN_AUDIO_VOLUME{{System::Main, "DSP", "Volume"}, 100};
 const Info<bool> MAIN_AUDIO_MUTED{{System::Main, "DSP", "Muted"}, false};
+const Info<bool> MAIN_AUDIO_MUTE_ON_TURBO{{System::Main, "DSP", "MuteTurbo"}, false};
```

Comment:
We can afford putting an extra word here to make the meaning clearer.

```suggestion
const Info<bool> MAIN_AUDIO_MUTE_ON_TURBO{{System::Main, "DSP", "MuteOnTurbo"}, false};
```